packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -78,7 +78,7 @@ class UpdatePackagesCommand extends FlutterCommand {
   }
 
   @override
-  bool get isExperimental => true;
+  bool get isExperimental => false;
```

Comment:
Isn't that the default? Should we just remove the override?

Reply:
Done